### PR TITLE
Desktop feature

### PR DESCRIPTION
**Pull request by JbirdL86**
In this branch we added some responsiveness to the portfolio project, this was made by taking a mobile-first approach. The main files modified in this feature were:

- style.css
- index.html
We also added some images for desktop resolution.

I hope you like my project so far!

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -483,29 +475,279 @@ header {
   border-radius: 10px;
 }
 
-@media only screen and (max-width: 700px) {
-  .form-container {
-    flex-wrap: wrap;
+@media only screen and (min-width: 600px) {
```
Comment (Davidosky007) [inline style.css]: Please, use media queries for the Desktop breakpoint in the CSS file.(min-width should be greater than 992px - 1100px), for example: @media screen and (min-width: 993px upward)  👍 read this [Link](https://responsivedesign.is/strategy/page-layout/defining-breakpoints/) for more info

File: style.css [hunk 2]
```diff
@@ -483,29 +475,279 @@ header {
...
-    text-align: center;
-    margin-bottom: 73px;
-    height: 216px;
-    margin-left: 0;
-    margin-right: 0;
-    overflow: hidden;
+    text-align: left;
+    margin-left: 135px;
+    margin-right: 24px;
+    max-width: 455px;
```
Comment (Davidosky007) [inline style.css:741]: I suggest you stick to the template design as much as possible. This is your design seems the webpage is overflowing 
Screenshot (37)

This is the Figma design template please compare and make the necessary adjustments
Screenshot (38)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @jchernandez87 @JbirdL86, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
CR3_gif_almost_there


 ## Required Changes ♻️
* I suggest you start your commit messages with capital letters read [This](https://www.theserverside.com/video/Follow-these-git-commit-message-guidelines) for further info

_Check the comments under the review and make the necessary changes ._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css [hunk 3]
```diff
@@ -483,29 +475,281 @@ header {
   border-radius: 10px;
 }
 
-@media only screen and (max-width: 700px) {
-  .form-container {
-    flex-wrap: wrap;
+@media only screen and (min-width: 993px) {
```
Comment (MahmoudBakr23) [inline style.css:478]: Great job implementing the changes requested by the earlier reviewer. However, there is always room for improvement.
**OPTIONAL**
It'd be nicer if you managed another media query for the break of elements in the medium-sized screen. Also, it seems your mobile version having an overflow from the beginning. So, it'd be great if you worked on that too.
Good luck and happy coding with the next.

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @JbirdL86 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**OPTIONAL**
Please update your live demo link

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**